Tensile/Common.py

```diff
@@ -997,7 +997,7 @@
     {"ScheduleLocalWrite":        [ 1 ] },
     {"ScheduleIterAlg":           [ 1 ] },
 
-    {"LdcEqualsLdd":              [ True ] },
+    {"LdcEqualsLdd":              [ False ] },
```

Comment:
Suggest to describe why we still keep it in PR.
Better to write down about how to remove it cleanly in future work.

Reply:
add

Reply:
add comment on declaration